back/lib/voting_method/voting.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module VotingMethod
+  class Voting < Base
+    def initialize(participation_context)
+      # Set the default voting term for the context
+      if participation_context.voting_term_singular_multiloc.blank? && participation_context.voting_term_plural_multiloc.blank?
+        participation_context.voting_term_singular_multiloc = CL2_SUPPORTED_LOCALES.index_with do |locale|
```

Comment:
This means that the default is persisted in the database. This means that it is impossible to update the default term for existing tenants. It's a big issue we have today with other data (topics, custom fields, statuses etc.). That's why we introduced dynamic multilocs for nav bar items. Unless we're certain we don't want to update the default term for existing tenants, I think we should consider implementing a dynamic multiloc.

Reply:
Have you got an example of dynamic multiloc? However, I do think it's highly unlikely that the default term for voting will change and actually within the context of a voting phase or project which will be time limited, it is probably beneficial that the term doesn't change whilst people are voting.

Reply:
You can find an example in `nav_bar_item.rb`. Although, I'm no longer up to date with latest changes, so the current behaviour may no longer be as originally intended.

Indeed in English I wouldn't expect this to change in the future. But we have many other languages, and I wouldn't be surprised if in one of these languages there are two synonymous words that both mean "vote". If at some point, we change the translation from one word to the other synonymous word, we'll have a small nightmare to identify which of the existing platforms need to be updated, especially if the translation changed before (we don't keep track of the history of a translation well).

Reply:
But this part is not blocking for me to approve the PR :)